ivy/functional/frontends/paddle/tensor/math.py

```diff
@@ -412,3 +412,17 @@ def any(x, axis=None, keepdim=False, name=None):
 @to_ivy_arrays_and_back
 def diff(x, n=1, axis=-1, prepend=None, append=None, name=None):
     return ivy.diff(x, n=n, axis=axis, prepend=prepend, append=append)
+
+
+@with_supported_dtypes(
+    {"2.5.0 and below": ("float32", "float64", "int32", "int64")}, "paddle"
+)
+@to_ivy_arrays_and_back
+def increment(x, value=1.0):
+    if x.size == 1:
```

Comment:
`x.size` returns a list. Maybe you forgot to take its length?

Reply:
I am a bit confused by this: if type in the terminal something like `a = ivy.array([3,4,5,6,7]` and then do `a.size` I just get 5, I want to check if the whole array contains exactly one element.

Reply:
@FelixHuyghe Why is this `if` condition needed anyways?

Reply:
The paddle function accepts only inputs with one element: 
`paddle.increment(x, value=1.0, name=None)
The API is usually used for control flow to increment the data of x by an amount value. Notice that the number of elements in x must be equal to 1.`
The problem is that ivy.increment_incplace accepts inputs with any shape, so I thought I check if the size is equal to one and if not I return an exception. However if I try to test this by setting the max_num_size parameter for x to something larger then 1, the tests fail and I get the exception I wrote myself. I was trying to test that when you provide an input array with a size larger thant 1 the function indeed raises an exception, just like the original Paddle function. But I don't think I am doing it in the correct way if you could help me :)

Reply:
> The paddle function accepts only inputs with one element: `paddle.increment(x, value=1.0, name=None) The API is usually used for control flow to increment the data of x by an amount value. Notice that the number of elements in x must be equal to 1.` The problem is that `ivy.increment_incplace` accepts inputs with any shape, so I thought I check if the size is equal to one and if not I return an exception. However if I try to test this by setting the `max_num_size` parameter for x to something larger then 1, the tests fail and I get the exception I wrote myself. I was trying to test that when you provide an input array with a size larger than 1 the function indeed raises an exception, just like the original Paddle function. But I don't think I am doing it in the correct way, if you could help me :) I was also wondering if in this case the frontend function should return the exact same exception as when paddle.increment is called directly. This is why in my previous commit I tried calling paddle.increment directly from the frontend, to get the exact same exception, but I understand this is not the way to go.

@FelixHuyghe You don't need to test for values that you already know don't fall into the acceptable range of values for this function. The unit test is required to provide complete coverage of an acceptable range of values and so only values that the original function should run fine on should be generated with the unit tests if that makes sense. :)